app/services/message_employee_matcher.rb

```diff
@@ -21,10 +21,9 @@ def day_count
   end
 
   def match_time(time_zone)
-    employee_current_hour = Time.current.in_time_zone(time_zone).hour
-    employee_current_minute = Time.current.in_time_zone(time_zone).min
+    employee_current_time = Time.current.in_time_zone(time_zone)
 
-    employee_current_hour == message.time_of_day.hour && employee_current_minute == message.time_of_day.min
+    employee_current_time.utc.strftime("%H%M%S%N") >= message.time_of_day.utc.strftime("%H%M%S%N")
```

Comment:
The problem with that in this case is that the date portion of the Time object will not match, nor do we care about it - we just want the time portion of the scheduled message (e.g., the scheduled message is set to go out at 12:00 PM) and compare that to just the current local time of the user.  Comparing the objects directly is going to factor in the date too, and the date in the case of the scheduled message happens to get filled in as 2000-01-01.